cmd/minikube/cmd/config/disable.go

```diff
@@ -38,11 +40,24 @@ var addonsDisableCmd = &cobra.Command{
 		if addon == "heapster" {
 			exit.Message(reason.AddonUnsupported, "The heapster addon is depreciated. please try to disable metrics-server instead")
 		}
-		err := addons.SetAndSave(ClusterFlagValue(), addon, "false")
+		cc, err := config.Load(ClusterFlagValue())
 		if err != nil {
-			exit.Error(reason.InternalAddonDisable, "disable failed", err)
+			exit.Error(reason.InternalAddonDisable, "loading profile", err)
```

Comment:
the reason should be something different, if there is not a reason for it we can create one

Reply:
Do we need a reason or just an error message instead?

Reply:
reason package is a way to help users quickly identify the Real Reason minikube failed... so it would generate a unique keyword for the error message
also when users collect metrics what type of failures minikube have...we can identify this as a Unique one...and measure how many times this specific err happens. (so we prefer as many unique error key words (reasons) for uniuqe reaons)

Reply:
Makes sense. Have a look at the updated code, I think it is good now.